### DISCUSSION TITLE
The model weights are heavy

### DISCUSSION THREAD

**Discussion by talhaanwarch** [autogluon/autogluon#1167] [Q&A]
I have also worked with PyTorch and TensorFlow and my experience is that their pre-trained models are not large and less than 20 mb, but in case of autogluon the size is around 200 mb. Is there a way to reduce it?

**Comment by MunumButt** [comment]
I would assume using a lighter preset would help

**Reply by talhaanwarch** [reply]
does autogluon save weights for all batch sizes, learning rates it tried?
